src/pipe/modules/i-raw/flat.mk

```diff
@@ -1,6 +1,9 @@
 RAWSPEED_I=../ext/rawspeed
 RAWSPEED_L=../built/ext/rawspeed
 MOD_CFLAGS=-std=c++17 -Wall -I$(RAWSPEED_I)/src/librawspeed/ -I$(RAWSPEED_L)/src/ -I$(RAWSPEED_I)/src/external/ $(shell pkg-config --cflags pugixml libjpeg)
+ifeq ($(shell uname),Darwin)
```

Comment:
again, probably better to have such things in a custom `bin/config.mk` and routed through somehow.